### ISSUE TITLE
Adapting to loupedeck-live ?

### ISSUE THREAD

**Issue by benkuper** [bitfocus/loupedeck-ct#7]
Hello,
I'm the developer of Chataigne : http://benjamin.kuperberg.fr/chataigne (C++)
I recently acquired a LoupeDeck live and the only relevant information I found about communicating directly to loupedeck devices is here.

I was wondering if adapting your reverse engineering to loupedeck live is imaginable to some extent ?
I would love to be able to integrate it the same way I did for StreamDeck (there is a github page exposing the communication protocol, no code dependency).
At least if there is a possibility to extract the communication protocol from your repo and expose it in this way 
 : https://gist.github.com/cliffrowley/d18a9c4569537b195f2b1eb6c68469e0
I think this would help people wanting to get their hands dirty and try to contribute to this effort.

I could spend some time trying to reverse it, but I have very little knowledge and skills in that field, however I would be up for some hacking session to understand a bit better and bring whatever I can to the table !

**Comment by latenitefilms** [comment]
This might also help:

https://github.com/CommandPost/CommandPost/blob/develop/src/extensions/hs/loupedeck/init.lua

**Comment by benkuper** [comment]
Well, yes !
Thanks @latenitefilms !!

**Comment by benkuper** [comment]
@latenitefilms Looking at your code, I would have few questions (using websocket, no libusb ?) if you don't mind. Is there a way to contact you without spamming this repo ?
btw your demoreel looks awesome :)

**Comment by latenitefilms** [comment]
Yep - feel free to [contact me](https://latenitefilms.com/contact/).

**Comment by Julusian** [comment]
There are other npm libraries for the other models such as https://www.npmjs.com/package/@loupedeck/node
Eventually that one will support the CT too, but it doesnt yet
